### PR TITLE
Make Malawi Aviation Data Available

### PR DESCRIPTION
**Pull request by espwx**
# The purpose of this Pull Request
To make available Malawi Aviation Data:
1. Create a Malawi Terrain File Bounding Box (MW.json)
2. Merge Waypoints Data with existing File (added Aerodromes, Navaids, Navigation Lights, Obstacles, Reporting Points)
3. Malawi Airspace file, including CTRs, TMAs, CTA, Danger, Restricted and Prohibited Airspace.

# Data Source
The official Malawi AIP last updated on 1 January 2011 an extract of which is available here:
https://github.com/espwx/aviation-mw

### CODE REVIEW

**Review by lordfolken — APPROVED** [review]

File: data/content/waypoint/country/MW.cup [hunk 1]
```diff
@@ -1,10 +1,81 @@
...
-"Blantyre Chileki",,MW,1540.783S,03458.050E,780.0m,5,100,2330.0m,118.100,"Flugplatz"
-"Chelinda Malawi",,MW,1033.000S,03348.000E,2365.0m,5,080,1290.0m,,"Flugplatz"
-"Chilumba Prv",,MW,1026.000S,03415.000E,491.0m,2,080,890.0m,,"Flugplatz"
-"Karonga",,MW,0957.300S,03353.750E,540.0m,5,140,1290.0m,118.100,"Flugplatz"
-"Lilongwe Kamuzu",,MW,1347.000S,03346.650E,1232.0m,5,140,3550.0m,118.700,"Flugplatz"
-"Mchinji",,MW,1348.000S,03254.000E,1189.0m,2,150,980.0m,,"Flugplatz"
-"Mzuzu",,MW,1126.883S,03400.867E,1256.0m,5,170,1290.0m,118.100,"Flugplatz"
-"Nchalo Sucoma",,MW,1616.000S,03455.000E,70.0m,2,160,740.0m,,"Flugplatz"
-"Ntchisi",,MW,1322.000S,03351.000E,1311.0m,2,150,920.0m,,"Flugplatz"
+name,code,country,,,elev,style,rwydir,rwylen,freq,desc
```
Comment (lordfolken) [inline data/content/waypoint/country/MW.cup]: Please remove this header

**Review by lordfolken — CHANGES_REQUESTED** [review]
Really remove the line please.

File: data/content/waypoint/country/MW.cup (hunk 1 above)
Comment (espwx) [reply data/content/waypoint/country/MW.cup]: OK, done that.

**Review by lordfolken — DISMISSED** [review]

File: data/content/waypoint/country/MW.cup [hunk 2]
```diff
@@ -1,10 +1,81 @@
...
+Chizumulu Island,,MW,1201.000S,03437.000E,,8,,,,"Light Marine, FLG(1) W EV 9 Sec, H24"
+Kambwe Lagoon,,MW,0953.000S,03357.000E,,8,,,,"Light Marine, FLG(1/2) W EV 4.5 Sec, H24"
+Karonga MBC,,MW,0956.783S,03354.917E,,8,,,,"Light HBN, FLG R, HN"
+Katenga Isle,,MW,1355.000S,03438.000E,,8,,,,"Light Marine, FLG(1) W EV 9 Sec, H24"
+Lilongwe MBC,,MW,1400.117S,03345.100E,,8,,,,"Light HBN, FLG R, HN"
+Mangochi MBC,,MW,1427.217S,03513.567E,,8,,,,"Light HBN, FLG R, HN"
+Mbenje Isle,,MW,1325.000S,03429.000E,,8,,,,"Light Marine, FLG(1/4) W EV 2.25 Sec, H24"
+Mzuzu MBC,,MW,1130.000S,03356.000E,,8,,,,"Light HBN, FLG R, HN"
+Namalenje Island,,MW,1344.000S,03438.000E,,8,,,,"Light Marine, FLG(1/2) W EV 4.5 Sec, H24"
+Ngumbe MBC,,MW,1542.083S,0351.650E,,8,,,,"Light HBN, FLG R, HN"
```
Comment (lordfolken) [inline data/content/waypoint/country/MW.cup]: The longitude here is missing a digit. "0351.650E"

Comment (espwx) [reply data/content/waypoint/country/MW.cup]: Done. Hopefully it works now.

**Review by lordfolken — APPROVED** [review]

**Pull request merged**